glambie/processing/path_handling.py

```diff
@@ -0,0 +1,109 @@
+from glambie.const.regions import RGIRegion
+from glambie.const.data_groups import GlambieDataGroup
+import os
+from pathlib import Path
+import datetime
+
+
+class OutputPathHandler():
+    """
+    Handles paths for Glambie processing intermediate results, plots and other outputs
+    """
+
+    def __init__(self, base_path: str):
+        """
+        Constructor
```

Comment:
Would change this to say what it does - here, it seems to create the base path if it does not exist already

Reply:
done